docker-compose.yml

```diff
@@ -70,8 +70,7 @@ services:
     ports:
       - "15432:5432"
     volumes:
-      - ./pg_data:/var/lib/pgsql/data
```

Comment:
Should we keep this? https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/rbac/rbac-pgsql.yaml#L98

Reply:
That may be the right path in e2e for the `postgresql-96-centos7` image. I observed in running the `postgres:9.6` image here that the pgdata path was incorrect, so I corrected it.

Reply:
Yeah, I noticed that with regard to the `postgresql-96-centos7` image in e2e, that is the correct path. I tried rebuilding my local with `postgres:9.6` and am still having an issue with this, though. @astrozzc do you see the same?